src/Apps/W1/Quality Management/app/src/Configuration/Grade/QltyGradeEvaluation.Codeunit.al

```diff
@@ -603,9 +597,9 @@ codeunit 20410 "Qlty. Grade Evaluation"
     /// <summary>
     /// OnBeforeEvaluateGrade gives an opportunity to change how a grade is evaluated.
     /// </summary>
-    /// <param name="TestQltyIGradeConditionConf">VAR record "Qlty. Grade Condition Config".</param>
-    /// <param name="FieldType">VAR enum "Qlty. Field Type".</param>
-    /// <param name="TestValue">VAR Text.</param>
+    /// <param name="TestQltyIGradeConditionConf">var Record "Qlty. Grade Condition Config".</param>
+    /// <param name="FieldType">var Rnum "Qlty. Field Type".</param>
+    /// <param name="TestValue">var Text.</param>
```

Comment:
These are not proper documentation. What are the parameters suppose to represent? Not the signature.